independent-publisher-connectors/Informa/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1976 @@
+              "properties": {
+                "campoCodificadoRespuesta": {
+                  "type": "object",
+                  "properties": {
+                    "valor": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "code value",
+                      "title": "code value"
+                    },
+                    "tablaDecodificacion": {
+                      "type": "string",
+                      "description": "Decoding table",
+                      "title": "Decoding table"
+                    },
+                    "literal": {
+                      "type": "string",
+                      "description": "literal"
+                    }
+                  },
+                  "description": "campoCodificadoRespuesta"
+                },
+                "datosPeticion": {
+                  "type": "object",
+                  "properties": {
+                    "productoSolicitado": {
+                      "type": "string",
+                      "description": "Requested product",
+                      "title": "Requested product"
+                    },
+                    "numeroVersionTablasInformacionComercial": {
+                      "type": "number",
+                      "format": "float",
+                      "description": "Comercial information table version number",
+                      "title": "Comercial information table version number"
+                    },
+                    "parametrosCliente": {
+                      "type": "object",
+                      "properties": {
+                        "idioma": {
+                          "type": "string",
+                          "description": "Language",
+                          "title": "Language"
+                        },
+                        "username": {
+                          "type": "string",
+                          "description": "Username",
+                          "title": "Username"
+                        },
+                        "formato": {
+                          "type": "string",
+                          "description": "format",
+                          "title": "format"
+                        },
+                        "cif": {
+                          "type": "string",
+                          "description": "Client Identifier",
+                          "title": "Client Identifier"
+                        }
+                      },
+                      "description": "parametrosCliente"
+                    }
+                  },
+                  "description": "datosPeticion"
+                },
+                "datosProducto": {
+                  "type": "object",
+                  "properties": {
+                    "informacionComercial": {
+                      "type": "object",
+                      "properties": {
+                        "identificacion": {
```

Comment:
Please correct the spelling "identificacion"

Reply:
I think I should use 'formato' which is what the API expects. Is in spanish.

Reply:
Kindly post your connector in English, this will be translated to other languages later.

Reply:
The API requires specific parameters, as indicated. This is a Spanish solution that provides public information about Spanish companies.